launcher/src/editor-configurations.ts

```diff
@@ -8,20 +8,16 @@
  * SPDX-License-Identifier: EPL-2.0
  ***********************************************************************/
 
-import * as k8s from '@kubernetes/client-node';
 import * as fs from './fs-extra.js';
-import { ProductJSON } from './product-json.js';
 import { mergeFirstWithSecond, parseJSON } from './json-utils.js';
+import { ProductJSON } from './product-json.js';
 
-const CONFIGMAP_NAME = 'vscode-editor-configurations';
+const CONFIGS_PATH = '/vscode-editor-configs';
```

Comment:
Could we apply `/checode/.config` ?
Just to not make the root directory messy.

Reply:
I tried to configure the path as
```
controller.devfile.io/mount-as: subpath
controller.devfile.io/mount-path: /checode/remote/data/configs
```
but faced a problem that it removes other files in the `/checode/remote/data`
so - I had to configure it to a place where it's safe and doesn't lead to bugs...